ui/raidboss/data/05-shb/dungeon/matoyas_relict.js

```diff
@@ -62,22 +63,21 @@ export default {
     },
     {
       id: 'Matoyas Nixie Crash-smash',
-      netRegex: NetRegexes.startsUsing({ id: '598F', source: 'Nixie' }),
-      netRegexDe: NetRegexes.startsUsing({ id: '598F', source: 'Nixchen' }),
-      netRegexFr: NetRegexes.startsUsing({ id: '598F', source: 'nixe' }),
-      netRegexJa: NetRegexes.startsUsing({ id: '598F', source: 'ノッケン' }),
-      netRegexCn: NetRegexes.startsUsing({ id: '598F', source: '水滴精' }),
-      netRegexKo: NetRegexes.startsUsing({ id: '598F', source: '뇌켄' }),
-      condition: Conditions.targetIsNotYou(),
-      alertText: (data, _, output) => output.avoidTether(),
+      netRegex: NetRegexes.headMarker({ id: '00E6' }),
+      alertText: (data, matches, output) => {
+        if (data.me === matches.target)
+          return output.tankBuster();
+        return output.avoidTether({ player: matches.target });
+      },
       outputStrings: {
+        tankBuster: Outputs.tankBuster,
         avoidTether: {
-          en: 'Avoid tank and tethers',
-          de: 'Weiche den Tank-Verbindungen aus',
-          fr: 'Évitez le tank et les liens',
-          ja: 'タンクと線から離れる',
-          cn: '远离坦克及其连线',
-          ko: '탱커와 선 피하기',
+          en: 'Avoid ${player} and tethers',
+          de: 'Weiche den ${player}-Verbindungen aus', // FIXME
+          fr: 'Évitez ${player} et les liens', // FIXME
+          ja: '${player}と線から離れる',
+          cn: '远离${player}及其连线',
+          ko: '${player}와 선 피하기', // FIXME
```

Comment:
```suggestion
          ja: '${player}と線から離れる',
          cn: '远离${player}及其连线',
```

I think in cases where you're not sure, it's maybe better to just remove it and have somebody re-translate it.

Reply:
Okay!